src/main/java/core/basesyntax/suplier/Supplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.suplier;
+
+import java.util.Random;
+
+public abstract class Supplier {
+    public static final Random RANDOM = new Random();
+    public static final int MIN_VALUE = 1;
+    public static final int MAX_VALUE = 20;
```

Comment:
good job on that class, lest improve it a bit :wink: 
let's not make random static and not name it as constant, because it does not quite fit in the dependency injection principle, for now, we will initialize similar objects like this, and only constants with types of primitives or wrappers will be static

Reply:
I made changes.